lib/client-api/src/story_store.js

```diff
@@ -282,7 +282,7 @@ export default class StoryStore extends EventEmitter {
     }
 
     const { story } = data;
-    return () => story();
+    return story;
```

Comment:
@ndelangen

So now getStory and getStoryWithContext do actually the same thing. Should we deprecate one of those two?